route/config/route_config_params.yaml

```diff
@@ -4,4 +4,7 @@ max_crosstrack_error: 25
 
 # Double: downtrack_range
 # Units: meter
-destination_downtrack_range: 0
\ No newline at end of file
+destination_downtrack_range: 0
+
+# Int: maximum cross track error counter
```

Comment:
I think something like "Maximum number of consecutive timesteps outside of the route allowable before triggering LEFT_ROUTE" or something similar would probably be preferable for this still. As is it leaves room for interpreting what a crosstrack error is and how/when they are counted.